pkg/hive/hive.go

```diff
@@ -299,17 +310,31 @@ func (s *Service) checkAndAddPeers(ctx context.Context, peers pb.Peers) {
 			ctx, cancel := context.WithTimeout(ctx, pingTimeout)
 			defer cancel()
 
-			start := time.Now()
-
 			// check if the underlay is usable by doing a raw ping using libp2p
-			if _, err := s.streamer.Ping(ctx, multiUnderlay); err != nil {
+			// check if node is running in browser, and if it is, ping wss only
```

Comment:
The comment refers to a particular situation, but the code is actually more general, trying to find any reachable underlay, which would work just fine in wss case as well. I think that the comment should be updated.